app/src/main/java/com/example/atheneum/fragments/AddBookFragment.java

```diff
@@ -132,17 +132,21 @@ public void saveNewBook() {
         Log.i("AddBook", "AddBook*** Save Button Pressed");
 
         if (allFieldsFilled()) {
+            Log.i("AddBook", "AddBook*** Fields checked, all filled");
+
             title = titleEditText.getText().toString();
             author = authorEditText.getText().toString();
             isbn = Long.parseLong(isbnEditText.getText().toString());
             desc = descEditText.getText().toString();
 
             // get user from Firebase auth
-            FirebaseAuth auth = FirebaseAuth.getInstance();
+//            FirebaseAuth auth = FirebaseAuth.getInstance();
             FirebaseUser firebaseUser = FirebaseAuth.getInstance().getCurrentUser();
             final FirebaseDatabase db = FirebaseDatabase.getInstance();
 
             if (firebaseUser != null) {
+                Log.i("AddBook", "AddBook*** not null firebaseuser");
```

Comment:
You could add a TAG variable for the log so you don't have to duplicate the "AddBook" part.

Reply:
Fixed